### PR TITLE
Fixes #32 - Missing license

### LINKED ISSUE #32: Missing LICENSE or COPYING
The license terms are missing. In the package.json it appears the code is licensed under the MIT license however the license is not included in the repository nor is it clearly labeled in the README.md or in any other files.

I would like to look at the code and potentially adapt some of it for use in another free/open source project.

### PR DESCRIPTION
**Pull request by locks**
@rwjblue make sure I got the © right

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2013-2015 Tilde, Inc. and contributors.
```
Comment (rwjblue) [inline LICENSE]: I'm not sure that anyone from Tilde has actually worked on this. 

Can you make it "Ember Core Team and contributors"?

Comment (locks) [inline LICENSE]: Will do, I wasn't quite sure what to put there.

**Comment by rwjblue** [comment]
Thank you!

**Comment by locks** [comment]
no problem ;)

**Pull request merged**